ESP3Packet.js

```diff
@@ -44,10 +44,12 @@ function ESP3Packet() {
 	this.header = {
 		dataLength: undefined,
 		optionalLength: undefined,
-		packetType: undefined
+		packetType: undefined,
+		raw: undefined
```

Comment:
I think this is unnecessary

just use following in Lines L28-L30 of [ESP3Parser.js](https://github.com/RafaelKa/node-serialport-enocean-parser/blob/d540e2ae846d3f9265bc7ca40656fb6e614cc701/ESP3Parser.js#L28-L30)

`
		for (var offset = 0; offset < buffer.length; offset++) {
			currentESP3Packet.raw.fill(buffer[offset], currenWholeOffsetOfPacket()); // current offset of whole Packet must be calculated. Don't know currently how handles JS addition of undifined vars ;)
			callbackForNextByte(buffer[offset]);
		}
`
and removo anything else from ESP3Parser.js

What do you thing about this proceedings?
so all is done in the same loop...